Makefile

```diff
@@ -0,0 +1,45 @@
+TEST=tests
+PDB_PATH=$(shell find /usr/lib{,64} -type f -executable -name pdb.py 2> /dev/null)
+DATA=$(shell echo $(MODULE) | sed 's/\(foreman\|katello\)_//; s/_/-/g')
```

Comment:
`make` has functions for [text substitution](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html), do we want to use them?

Reply:
Here's what it would look like (perhaps there's a simpler way):

```
DATA=$(substr, _,-,$(filter-out katello_, $(filter-out foreman_, $(MODULE)))
```

I don't know about you, but I don't like it.